### ISSUE TITLE
a11y: Identical label for form fields - `banners` settings area in Settings

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#14092]
**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

Identical label for form fields - currently the screen reader announces every UI element without providing the context of the area (ex: text for header banner is announced as `text`, same for footer banner) so the user will have much more difficulties navigating through the UI.

**To Reproduce**

**Result**
Identical label is specified for the form fields resulting in screen reader users might find it difficult to understand the purpose of the form fields effectively.

**Expected Result**

Screen reader should announce unique and descriptive label for checkboxes and radio buttons.

**Screenshots**

**Additional context**

**Comment by izaac** [comment]
Rancher version: `v2.12-9d8a81e5882a1c435fd4b5f68428e59dd69d73d1-head`
Rancher UI version (dashboard): `master f206b54`

I've navigated through all the banner settings using the Windows Narrator and a Chrome accessibility extension. 
Looks consistent, the user can tell exactly which element they are on and their state. ✅
